sovrin_client/cli/command.py

```diff
@@ -67,13 +75,13 @@
     examples="send proofreq Over-21 to JaneDo")
 
 showFileCmd = Command(
-    id="show file",
+    id="show",
```

Comment:
Yes, we have other show commands such as 'show claim', 'show link', and they have corresponding IDs ('show claim', 'show link').
But currently 'show' command means show file. There is no 'file' suffix there. If we need to fix it, we need a bug ticket.
BTW, there was a lot of code duplication and usage of string constants to identify and distinguish  commands. I made a change to always use command's ID for this purposes.